### ISSUE TITLE
Maybe fast, but not correct: charset_normalizer fails to detect UTF-8 *with small content*

### ISSUE THREAD

**Issue by vstinner** [Ousret/charset_normalizer#12]
I made a quick test:

```
vstinner@apu$ /bin/cat bla
héllo world!

vstinner@apu$ hexdump -C bla
00000000  68 c3 a9 6c 6c 6f 20 77  6f 72 6c 64 21 0a        |h..llo world!.|
0000000e

vstinner@apu$ env/bin/python -m charset_normalizer.cli.normalizer bla
+----------+----------+----------+----------------------------------------+-------+-----------+
| Filename | Encoding | Language |               Alphabets                | Chaos | Coherence |
+----------+----------+----------+----------------------------------------+-------+-----------+
|   bla    |   big5   | Unknown  | CJK Unified Ideographs and Basic Latin | 0.0 % |   0.0 %   |
+----------+----------+----------+----------------------------------------+-------+-----------+

vstinner@apu$ file bla
bla: UTF-8 Unicode text
```

charset_normalizer is wrong: the encoding is UTF-8, not big5.

```
vstinner@apu$ python3
Python 3.7.4 (default, Jul  9 2019, 16:32:37) 
>>> data=open("bla", "rb").read()
>>> data
b'h\xc3\xa9llo world!\n'

>>> print(ascii(data.decode('utf8')))  # expected result
'h\xe9llo world!\n'
>>> print(ascii(data.decode('big5')))  # wrong
'h\u77c7llo world!\n'
```

**Comment by Ousret** [comment]
Hi,

Indeed, small file are not our strong suit. I'm looking to improve this aspect.

**Comment by Ousret** [comment]
The real title of this issue is : "charset normalizer struggle with small content"

**Comment by Ousret** [comment]
```python
k = 'héllo world! héllo world! héllo world! héllo world!'.encode('utf_8')
from charset_normalizer.legacy import detect
detect(k)
```

encoding = utf_8

**Comment by Ousret** [comment]
Fixed with #14
